build.sbt

```diff
@@ -40,6 +40,9 @@ lazy val freestyle = (crossProject in file("freestyle")).
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-free" % "0.9.0",
       "org.scala-lang" % "scala-reflect" % scalaVersion.value,
+      "io.monix" %%% "monix-eval" % "2.2.1" % "test",
+      "io.monix" %%% "monix-cats" % "2.2.1" % "test",
```

Comment:
Test dependencies are not much of a problem, since they are intended not to be packaged in a release. They only concern those who are developing the library. 

For what is worth, a few other things may be scrapped from the core. Another day.